webpack.config.js

```diff
@@ -130,7 +130,13 @@ module.exports = function (env) {
           use: ExtractTextPlugin.extract({
             fallback: 'style-loader',
             use: [
-              'css-loader',
+              {
+                loader: 'css-loader',
+                options: {
+                  modules: true,
```

Comment:
Can you try css-loader 0.14.5 but instead of `modules: true` type `module: true`? You could add both options just in case. 0.14.5 is MUCH faster.